ReactJS/lesson_1/HW_2.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>calculateArea</title>
+</head>
+<body>
+<!--
+  Написать функцию calculateArea, которая будет принимать параметры, для вычисления площади (можете выбрать какую то
+  конкретную фигуру, а можете, основываясь на переданных параметрах, выполнять требуемый алгоритм вычисления площади
+  для переданной в параметрах фигуры) и возвращать объект вида: { area, figure, input }, где area - вычисленная площадь,
+  figure - название фигуры, для которой вычислялась площадь, input - входные параметры,
+  по которым было произведено вычисление.
+-->
+
+<script>
+  'use strict';
+
+  const calculateArea = (...params) => {
+    params = params[0]
```

Comment:
```suggestion
    params = params
```
нет смысла в массиве, можно просто `string` оставить:)